.claude/skills/chain-integration-crew/skill.md

```
@@ -0,0 +1,196 @@
+- Leverages the existing `chain-integration` skill
+- Handles ChainAdapter, CAIP constants, feature flags
+- Web-side chain support (portfolio, send, receive)
+
+### Swapper Agent (if applicable)
+- Leverages the existing `swapper-integration` skill
+- Implements swapper support for the new chain
+- Quote fetching, transaction building, status tracking
+
+## Execution Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)